public/app/core/services/backend_srv.ts

```diff
@@ -539,7 +539,9 @@ export class BackendSrv implements BackendService {
       queryParams.set('accesscontrol', 'true');
     }
 
-    return this.get<FolderDTO>(`/api/folders/${uid}?${queryParams.toString()}`);
+    return this.get<FolderDTO>(`/api/folders/${uid}?${queryParams.toString()}`, undefined, undefined, {
+      showErrorAlert: false,
```

Comment:
Would it make sense to add a parameter to `getFolderByUid` that sets this instead (which defaults to true)? This function is being used in places where seeing the error might be relevant

Reply:
great minds think alike 🧠